extension/page-detect.js

```diff
@@ -58,6 +60,8 @@ window.pageDetect = (() => {
 		return isRepo() && blobPattern.test(location.href);
 	};
 
+	const hasCommentForm = () => isGist() || (isRepo() && (isIssue() || isNewIssue() || (isPR() && !isPRFiles()) || isCompare()));
```

Comment:
I meant, can we find a CSS class(es) that identifies the comment form, instead of assuming it's in all these places?